### ISSUE TITLE
Under the same validation set, the evaluation results of the EMA model after loading are inconsistent with those during training.

### ISSUE THREAD

**Issue by FriedaSmith** [baaivision/EVA#133]
Hello, I use `EVA-02/asuka/run_class_finetuning.py` to fine-tune an image recognition task. After training the model, I evaluated it by loading the trained model. When loading the` mp_rank_00_model_states.pt` under the `checkpoint-best`, the results on the validation set (the validation set is the same as the validation set during training) are The evaluation results during training are the same. However, when I load the model under the` checkpoint-best-ema`, the accuracy obtained is much smaller than the evaluation results during training  using EMA. What is the problem?

**Comment by FriedaSmith** [comment]
The problem has been resolved.
